### ISSUE TITLE
Missing dependency: dom-accessibility-api

### ISSUE THREAD

**Issue by tomi77** [testing-library/jasmine-dom#80]
- `jasmine-dom` version: 1.3.2
- `node` version: 14
- `npm` (or `yarn`) version: 6.14.17

### Relevant code or config:

```js
var your => (code) => here;
```

### What happened:

ERROR in ./node_modules/@testing-library/jasmine-dom/dist/toHaveAccessibleName.js 7:27-59
Module not found: Error: Can't resolve 'dom-accessibility-api'

### Reproduction:

import JasmineDOM from '@testing-library/jasmine-dom';

### Suggested solution:

add dom-accessibility-api to dependencies

**Comment by oriSomething** [comment]
Make sure you installed dependencies properly. I checked it locally and confirm it works

**Comment by tomi77** [comment]
Yes, I'm sure. Look at https://github.com/tomi77/jasmine-dom-bug

```
npm install
npm test


> test@1.0.0 test
> jasmine

Error: Cannot find module 'dom-accessibility-api'
Require stack:
- /Users/tomaszrup/tmp/test/node_modules/@testing-library/jasmine-dom/dist/toHaveAccessibleDescription.js
- /Users/tomaszrup/tmp/test/node_modules/@testing-library/jasmine-dom/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1051:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/tomaszrup/tmp/test/node_modules/@testing-library/jasmine-dom/dist/toHaveAccessibleDescription.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1149:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1203:10)
    at Module.load (node:internal/modules/cjs/loader:1027:32)
    at Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1051:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/tomaszrup/tmp/test/node_modules/@testing-library/jasmine-dom/dist/toHaveAccessibleDescription.js',
    '/Users/tomaszrup/tmp/test/node_modules/@testing-library/jasmine-dom/dist/index.js'
  ]
}
```

**Comment by oriSomething** [comment]
You right. It was hoisted by other packages. It seems also in the repo I use at work. Anyway I've made a PR https://github.com/testing-library/jasmine-dom/pull/81
When @brrianalexis will see this. I guess he'll release a new version. It can take a few days 🙏
